### PR TITLE
[DOCS] Node

### LINKED ISSUE #13270: Calls to set_process_input are pointless before node is ready
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Any OS, Godot master (6086252f66ac185b97b0580352383e4b068b9fe5)

**Issue description:**

Not sure if it's a bug or by design, but calls to set_process_input and other similar functions before a node is added to the scene get "overridden" after the node is ready. A node sets process_input flag within its "ready" handler based on the presence of the corresponding process function. So if you set the flag before the node is ready, your value gets overridden. This doesn't seem immediately obvious/intuitive.

### PR DESCRIPTION
**Pull request by mhilbrunner**
Closes #13270.

### CODE REVIEW

File: doc/classes/Node.xml [hunk 1]
```diff
@@ -63,7 +63,8 @@
 			</argument>
 			<description>
 				Called during the processing step of the main loop. Processing happens at every frame and as fast as possible, so the [code]delta[/code] time since the previous frame is not constant.
-				It is only called if processing has been enabled with [method set_process].
+				It is only called if processing is enabled, which is done automatically if this method is overriden,
+				and can be toggled with [method set_process].
```
Comment (akien-mga) [inline doc/classes/Node.xml]: Please don't use a line break here, as 1 line == 1 paragraph in the processed output.

**Comment by akien-mga** [comment]
Would be good to document the same thing for `_physics_process` and `_input` which behave the same.

**Comment by mhilbrunner** [comment]
Will update this and https://github.com/godotengine/godot-docs/pull/617 with changes for `_physics_process` and `_input`.

**Comment by mhilbrunner** [comment]
Should be good now, if I didn't miss anything. Added `_physics_process` and `_input` stuff, and a few of the missing methods and constants.

File: doc/classes/Node.xml [hunk 2]
```diff
@@ -769,16 +776,22 @@
 	</signals>
 	<constants>
 		<constant name="NOTIFICATION_ENTER_TREE" value="10">
+			Notification received when the node enters a [SceneTree].
 		</constant>
-		<constant name="NOTIFICATION_EXIT_TREE" value="11">
+		<constant name="NOTIFICATION_EXIT_TREE" value="11">#
```
Comment (TwistedTwigleg) [inline doc/classes/Node.xml]: `#` should be removed, since it does nothing (to the best of my knowledge)

File: doc/classes/Node.xml [hunk 3]
```diff
@@ -769,16 +776,22 @@
...
+			Notification received when the node enters a [SceneTree].
```
Comment (TwistedTwigleg) [inline doc/classes/Node.xml]: "...node enters..." -> "...node exits..."

File: doc/classes/Node.xml [hunk 4]
```diff
@@ -343,7 +347,7 @@
 			<argument index="0" name="node" type="Node">
 			</argument>
 			<description>
-				Return [i]true[/i] if the "node" argument is a direct or indirect child of the current node, otherwise return [i]false[/i].
+				Returns [code]true[code] if the "node" argument is a direct or indirect child of the current node, otherwise return [code]false[code].
```
Comment (TwistedTwigleg) [inline doc/classes/Node.xml]: A couple of the end cap tags need to be changed from [code] to [/code]

**Review by TwistedTwigleg — COMMENTED** [review]
Looks good! :+1:  I found a few places that need a little correction, but other than those places it all looks great!

**Comment by mhilbrunner** [comment]
Fixed the things found by @TwistedTwigleg, thanks for looking over it :)

**Pull request merged**